scripts/git-hooks/pre-commit-minimum

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+##
+# Automatically run tests when commiting and prevent committing if tests fail.
+# Intended for installation into local .git/hooks folder so it runs on the
+# developers machine. Tests are run locally and expect the developer's machine
+# to have PHP and PHPCS installed. Designed to run very quickly.
+#
+# $ cd .git/hooks; ln -s ../../scripts/git-hooks/pre-commit-minimum pre-commit
+##
+
+# Really fast check if syntax is at all parseable
+for FILE in $(git diff --cached --name-only); do
+if [[ "$FILE" =~ \.php$ ]]; then
+    php -l "$FILE" 1> /dev/null
+    if [[ $? -ne 0 ]]; then
+        echo -e "\e[1;33mAborting commit: PHP code contains syntax errors\e[0m" >&2
+        exit 1
+    fi
+fi
+done
+
+# Slower test if file follows WordPress coding standards (core + extra)
+# https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards
+for FILE in $(git diff --cached --name-only); do
+if [[ "$FILE" =~ \.php$ ]]; then
+    echo -n "Running phpcs "
+    phpcs --standard=phpcs.xml "$FILE"
```

Comment:
There is no phpcs.xml standard file in the project root directory:

```
$ git commit -am 'Test'
Running phpcs ERROR: the "phpcs.xml" coding standard is not installed. The installed coding standards are MySource, PEAR, PSR1, PSR12, PSR2, Squiz, Zend, Security, WordPress, WordPress-Core, WordPress-Docs and WordPress-Extra
\e[1;33mAborting commit: PHP code violates coding standards\e[0m
```
If this is supposed to be theme/plugin-specific, should the existence of such a file be checked?